pennylane/transforms/mitigate.py

```diff
@@ -215,6 +215,12 @@ def qfunc(op):
     # Generate base_circuit without measurements
     # Treat all circuits as lists of operations, build new tape in the end
     base_ops = circuit.operations
+    if any((isinstance(op, qml.operation.Channel) for op in base_ops)):
+        raise ValueError(
+            "Circuits containing quantum channels cannot be folded with mitigate_with_zne. "
+            "To use zero-noise extrapolation on the circuit with channel noise, "
+            "consider adding the noise on the device rather than the circuit."
```

Comment:
```suggestion
            "please include the noise on the device rather than the circuit."
```

Reply:
"consider" seems optional, but the user "has to" add the noise to the device, correct?